ts/Core/Series/Point.ts

```diff
@@ -1513,7 +1513,11 @@ class Point {
                                         markerAttribs.x,
                                         markerAttribs.y,
                                         markerAttribs.width,
-                                        markerAttribs.height
+                                        markerAttribs.height,
+                                        extend(
```

Comment:
Use `merge`. Unlike `extend` it will not change `markerOptions` in-place by default.